src/utils/arweave.ts

```diff
@@ -34,3 +36,16 @@ export const pruneTags = (
     }): tag is { name: string; value: string } => tag.value !== undefined,
   );
 };
+
+export const getCurrentBlockUnixTimestamp = async (
```

Comment:
nit: this returns a js timestamp, not unix

Reply:
[fa29090](https://github.com/ar-io/ar-io-sdk/pull/249/commits/fa290906d9ca5c02fd43cf7eda43f708e8792694)